rexmex/metrics/classification.py

```diff
@@ -339,13 +451,23 @@ def prevalence_threshold(y_true: np.array, y_score: np.array) -> float:
         y_score (array-like):  An N x 1 array of predicted values.
     Returns:
         pthr (float): The prevalence threshold value.
+
+    .. seealso::
+
+        - https://en.wikipedia.org/wiki/Sensitivity_and_specificity#Prevalence_threshold
+        - https://dx.doi.org/10.1371%2Fjournal.pone.0240215
     """
     fpr = false_positive_rate(y_true, y_score)
     tpr = true_positive_rate(y_true, y_score)
     pthr = (fpr ** 0.5) / (fpr ** 0.5 + tpr ** 0.5)
     return pthr
 
 
+@classifications.annotate(  # TODO
```

Comment:
help wanted - I'm not really sure what the bounds for this are nor whether bigger is better